software/elasticsearch/egg-elasticsearch.json

```diff
@@ -0,0 +1,41 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-07-24T15:19:46-05:00",
+    "name": "Elasticsearch",
+    "author": "dominic@stilma.net",
+    "description": null,
+    "features": null,
+    "images": [
+        "quay.io\/pterodactyl\/core:java-11"
+    ],
+    "file_denylist": [],
+    "startup": ".\/bin\/elasticsearch -E network.host=0.0.0.0 -E http.port={{SERVER_PORT}} -E discovery.seed_hosts={{DISCOVERY_SEED_HOSTS}}",
+    "config": {
+        "files": "{}",
+        "startup": "{\r\n    \"done\": \"started\"\r\n}",
+        "logs": "{}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n# Elasticsearch Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\n\r\n## Install Requirements\r\n\r\napt update\r\napt install -y jq ca-certificates gnupg2 wget tar curl\r\n\r\ncd \/mnt\/server\r\n\r\n## Install Elasticsearch\r\nwget https:\/\/artifacts.elastic.co\/downloads\/elasticsearch\/elasticsearch-7.13.4-linux-x86_64.tar.gz\r\ntar -xzf elasticsearch-7.13.4-linux-x86_64.tar.gz\r\nmv elasticsearch-7.13.4\/* .\r\nrm elasticsearch-7.13.4-linux-x86_64.tar.gz\r\nrm -R elasticsearch-7.13.4\r\n\r\n## Finish command\r\necho \"All done! Just start it up :)\"",
```

Comment:
Hard-coding a single version is not a great idea. It will be outdated very quickly with constant Elastic updates. Make the version a variable that users can provide, even better would be to fetch the latest versions and default to that

Reply:
I added a variable which you can put in the version and it gets the latest, as i'm not smart enough to fetch the latest :(